tests/codemods/test_django_receiver_on_top.py

```diff
@@ -0,0 +1,71 @@
+from core_codemods.django_receiver_on_top import DjangoReceiverOnTop
+from tests.codemods.base_codemod_test import BaseCodemodTest
+from textwrap import dedent
+
+
+class TestDjangoReceiverOnTop(BaseCodemodTest):
```

Comment:
It would be good to have a test case with a different `receiver` import where no change is made.